assets/launcher/lang/sklmessages_cs_CZ.properties

```diff
@@ -55,40 +55,40 @@ editor.name=Název Profilu
 editor.copy=Kopie %s
 editor.gamedir=Místo uložení hry
 editor.resolution=Rozlišení
-editor.visibility=Viditelnost Launcheru
-editor.hide=Schovat launcher a znovu otevřít po ukončení hry
-editor.close=Ukončit launcher když se nastartuje hra
-editor.keep=Nechat launcher otevřený
```

Comment:
Is it necessary to translate the word "launcher"? In my opinion, not translating it would be the better solution. It also refers to the name "SKlauncher".